src/app/cli/src/commands/system_info_command.rs

```diff
@@ -161,6 +147,11 @@ impl BuildInfo {
             cargo_target_triple: option_env!("VERGEN_CARGO_TARGET_TRIPLE"),
             cargo_features: option_env!("VERGEN_CARGO_FEATURES"),
             cargo_opt_level: option_env!("VERGEN_CARGO_OPT_LEVEL"),
+            workspace_dir: WorkspaceService::find_workspace()
+                .root_dir
+                .to_str()
+                .map(String::from),
+            container_version: container_version_output,
```

Comment:
This shouldn't be a part of `BuildInfo` - this struct is meant to carry information about how `kamu` binary was compiled.